src/components/WalletConnect/SignMessageModal.js

```diff
@@ -7,61 +7,74 @@
 
 import React from 'react';
 import { useDispatch } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
 import { t } from 'ttag';
 import { StyleSheet, Text } from 'react-native';
-import ApproveRejectModal from './ApproveRejectModal';
 import { COLORS } from '../../styles/themes';
+import { ModalBase } from '../ModalBase';
+import { WarnDisclaimer } from './WarnDisclaimer';
+import { walletConnectReject } from '../../actions';
 
-const modalStyle = StyleSheet.create({
-  signMessageText: {
-    backgroundColor: COLORS.textColorShadowLighter,
-    width: '100%',
-    height: 100,
-    borderRadius: 15,
-    padding: 8,
-    marginBottom: 12,
-    marginTop: 12,
+const styles = StyleSheet.create({
+  body: {
+    paddingBottom: 24,
+  },
+  text: {
+    fontSize: 14,
+    lineHeight: 20,
+  },
+  bold: {
+    fontWeight: 'bold',
+  },
+  selectionContainer: {
+    borderRadius: 8,
+    paddingVertical: 8,
+    paddingHorizontal: 16,
+    backgroundColor: COLORS.freeze100,
   },
 });
 
 export default ({
-  onAcceptAction,
-  onRejectAction,
   onDismiss,
   data,
-  baseStyles,
```

Comment:
Why did you remove this? It's still being passed to the component

Reply:
Not being used, removed it.

We can remove it from WalletConnectModal when we refactor the Connect modal to use the new modal format Alex created